common/s2n-codec/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "s2n-codec"
-version = "0.35.1"
+version = "0.36.0"
```

Comment:
I'd prefer to have a specific commit to bump, just so we can point the tag at that commit

Reply:
https://github.com/aws/s2n-quic/pull/2180